operations/helm/charts/mimir-distributed/values.yaml

```
@@ -1286,6 +1286,17 @@ querier:
     minReplicaCount: 1
     maxReplicaCount: 10
     querySchedulerInflightRequestsThreshold: 12
+    # -- predictiveScalingEnabled scales up the querier based on the inflight requests a week ago (6d23h30m).
```

I think this is more accurate now that the time range is configurable:

```suggestion
    # -- predictiveScalingEnabled scales up the querier based on the inflight requests in the past.
```